apps/full-site-editing/package.json

```diff
@@ -81,6 +81,7 @@
 		"classnames": "2.2.6",
 		"lodash": "*",
 		"moment": "*",
-		"newspack-blocks": "github:Automattic/newspack-blocks#v1.2.1"
+		"newspack-blocks": "github:Automattic/newspack-blocks",
```

Comment:
removing the version from here has caused the CI to silently generate a faulty build where it has this definition with no version number:

```
// Autogenerated by apps/full-site-editing/bin/sync-newspack-blocks.sh.
define( 'NEWSPACK_BLOCKS__VERSION', '' );
// End autogenerated area.
```

Reply:
I removed the version in an attempt to make the CI integration work with the latest changes that are on master. I should have done a better job documenting that in the commit message.
And now that I look at it I realize I should have tried `master`.

Reply:
`#master` might just work! It will probably still generate `define( 'NEWSPACK_BLOCKS__VERSION', 'master' );`

that's a bit better than being empty but I wonder if that's a good dev workflow. Let's do whatever gets us working block and then figure out how to update our processes and scripts, if needed.

Reply:
> Let's do whatever gets us working block

That's really the intention here. Before we can launch this block all upstream improvements will have to be shipped. Then we can switch this to that new version